fenix/app/src/main/java/org/mozilla/fenix/translations/automatic/AutomaticTranslationItem.kt

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations.automatic
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.mozilla.fenix.R
+import java.util.Locale
+
+/**
+ * AutomaticTranslationItem that will appear on Automatic Translation screen.
+ *
+ * @property languageLocale The text that will appear in the list.
+ * @property automaticTranslationOption The option that the user selected.
+ */
+@Parcelize
+data class AutomaticTranslationItem(
+    val languageLocale: Locale,
+    val automaticTranslationOption: AutomaticTranslationOption,
+) : Parcelable
+
+/**
+ * AutomaticTranslationOption for a language.
+ *
+ * @property titleId The string id title of the option.
+ * @property summaryId The string id summary of the option.
+ */
+@Parcelize
+sealed class AutomaticTranslationOption(
+    open val titleId: Int,
+    open val summaryId: List<Int>,
+) : Parcelable {
+
+    /**
+     * The app will offer to translate sites in the selected language.
+     */
+    data class OfferToTranslate(
```

Comment:
Nice descriptions on this class! I think this class works just like we need it to for integration. 

Only one thing to keep in mind is we will probably will need to add  [`LanguageSetting`](https://searchfox.org/mozilla-central/rev/a9cb8b07d4cf4290e693bc31ba23412d0f042cb8/mobile/android/geckoview/src/main/java/org/mozilla/geckoview/TranslationsController.java#552) down the road for the final call to commit the change. Thought I'd mention it in-case you see it being an issue/something to think about.